services/QuillLMS/app/models/activity_session.rb

```diff
@@ -59,6 +59,10 @@ class StudentNotAssignedActivityError < StandardError; end
   NOT_YET_PROFICIENT = 'Not yet proficient'
   PROFICIENT = 'Proficient'
```

Comment:
Do we need to remove the old constants, or are they still in use anywhere?

Reply:
They're still in use, that'll be part of my follow-up.